src/prepare/compile.rs

```diff
@@ -0,0 +1,1239 @@
+#[allow(unused_imports)]
+use alloc::prelude::v1::*;
+
+use parity_wasm::elements::{BlockType, FuncBody, Instruction};
+
+use validation::func::{
+    require_label, top_label, BlockFrame, FunctionValidationContext, StackValueType, StartedWith,
+};
+use validation::stack::StackWithLimit;
+use validation::util::Locals;
+use validation::{Error, FuncValidator};
+
+use isa;
+
+/// Type of block frame.
+#[derive(Debug, Clone, Copy, PartialEq)]
```

Comment:
Why `PartialEq`?

Reply:
It seems it is used to be used, but we can remove it now for sure!